.github/workflows/check.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: check
+on: [push, pull_request]
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```

Comment:
I thought we'd standardised on 3.8?

Reply:
If we really want to use 3.8-specific syntax we can run pyflakes on 3.8. But I don't imagine we will be using any syntax introduced in 3.8.